doh-server/server.go

```diff
@@ -34,6 +34,10 @@ import (
 	"strings"
 	"time"
 
+	"crypto/tls"
+	"crypto/x509"
+	"io/ioutil"
+
```

Comment:
Why don't you reorganize your imports?
If you write code with a decent IDE, it should do this automatically for you.

Reply:
I am not writting go very often and don't know the etiquette, it seems go fmt is not performing it. Will try to find how to do it.

Reply:
1. For every standard library imports, sort them alphabetically.
2. Then have an empty line.
3. Then put every third-party libraries, sort them alphabetically.
4. Then have an empty line.
5. If you have any relative path imports (uncommon since Go 1.11) or something else, you may put them here.